controller_manager/scripts/controller_group

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# Provides quick access to the services exposed by controller_manager_msgs
+import rospy
+from argparse import ArgumentParser
+from controller_manager import controller_manager_interface
+from controller_manager_msgs.srv import ListControllers, ListControllersRequest
+
+
+class Groups(object):
+    @staticmethod
+    def list(args):
+        groups = rospy.get_param('/controller_groups', {})
+        print('Number of groups: {}'.format(len(groups)))
+        for group, controllers in groups.iteritems():
```

Comment:
For py3 friendliness, prefer just `items`. For such a small list, the savings from the iterator are extremely small anyway.

Reply:
Sure. Also I need sorted() for python 2 to print stable output